dashboard/src/components/widget/loader/che-bottom-bar.styl

```diff
@@ -0,0 +1,16 @@
+$link-color = #a5a5a5
```

Comment:
We have already one $link-color in  che-colors.styl. And they are different. I guess we should change this name.